pkgs/by-name/py/pyditz/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pyditz";
+  version = "0.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-2gNlrpBk4wxKJ1JvsNeoAv2lyGUc2mmQ0Xvn7eiaJVE=";
+  };
+
+  nativeBuildInputs = with python3Packages; [ setuptools-scm ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pyyaml
+    six
+    jinja2
+    cerberus
+  ];
+
+  meta = {
+    homepage = "https://pypi.org/project/pyditz/";
```

Opinionated change...

```suggestion
    homepage = "https://hg.sr.ht/~zondo/pyditz";
```